rotkehlchen/data_migrations/migrations/migration_10.py

```diff
@@ -22,11 +21,6 @@ def data_migration_10(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgress
     # steps are: ethereum accounts + 4 (potentially write to db + updating spam assets + polygon rpc + new round msg)  # noqa: E501
     progress_handler.set_total_steps(len(rotki.chains_aggregator.accounts.eth) + 4)
 
-    # Check updates for spam assets. This happens before accounts detection to avoid
```

Comment:
This is completely removed without a replacement. It's not in `update_data_and_detect_accounts`

Reply:
no it's in that functionfrom before. Resolving